.travis.yml

```diff
@@ -4,6 +4,7 @@ cache: bundler
 sudo: false
 
 rvm:
+  - 2.2
```

Comment:
it's best to use 2.2.7 here so you get the latest vs. the rvm alias for 2.2

Reply:
@tas50 thanks, updated